diff --git a/packages/common/src/keys.ts b/packages/common/src/keys.ts
index 59fe05a8..44a347a8 100644
--- a/packages/common/src/keys.ts
+++ b/packages/common/src/keys.ts
@@ -59,6 +59,7 @@ export const KEYS = {
   G: "g",
   H: "h",
   I: "i",
+  J: "j",
   L: "l",
   N: "n",
   O: "o",
diff --git a/packages/excalidraw/components/HelpDialog.tsx b/packages/excalidraw/components/HelpDialog.tsx
index 4f373c27..494830b9 100644
--- a/packages/excalidraw/components/HelpDialog.tsx
+++ b/packages/excalidraw/components/HelpDialog.tsx
@@ -157,7 +157,7 @@ export const HelpDialog = ({ onClose }: { onClose?: () => void }) => {
             />
             <Shortcut
               label={t("toolBar.diamond")}
-              shortcuts={[KEYS.D, KEYS["3"]]}
+              shortcuts={[KEYS.J, KEYS["3"]]}
             />
             <Shortcut
               label={t("toolBar.ellipse")}
diff --git a/packages/excalidraw/components/Tools.tsx b/packages/excalidraw/components/Tools.tsx
index 25e60c33..d4e0f06b 100644
--- a/packages/excalidraw/components/Tools.tsx
+++ b/packages/excalidraw/components/Tools.tsx
@@ -86,7 +86,7 @@ export const TOOLS = defineTools({
   },
   diamond: {
     icon: DiamondIcon,
-    letterKey: KEYS.D,
+    letterKey: KEYS.J,
     numericKey: KEYS["3"],
     fillable: true,
   },
diff --git a/packages/excalidraw/tests/__snapshots__/regressionTests.test.tsx.snap b/packages/excalidraw/tests/__snapshots__/regressionTests.test.tsx.snap
index b6d7b5ca..bc43aade 100644
--- a/packages/excalidraw/tests/__snapshots__/regressionTests.test.tsx.snap
+++ b/packages/excalidraw/tests/__snapshots__/regressionTests.test.tsx.snap
@@ -9942,7 +9942,7 @@ exports[`regression tests > key a selects arrow tool > [end of test] undo stack
 ]
 `;
 
-exports[`regression tests > key d selects diamond tool > [end of test] appState 1`] = `
+exports[`regression tests > key j selects diamond tool > [end of test] appState 1`] = `
 {
   "activeEmbeddable": null,
   "activeLockedId": null,
@@ -10072,13 +10072,13 @@ exports[`regression tests > key d selects diamond tool > [end of test] appState
 }
 `;
 
-exports[`regression tests > key d selects diamond tool > [end of test] number of elements 1`] = `0`;
+exports[`regression tests > key j selects diamond tool > [end of test] number of elements 1`] = `0`;
 
-exports[`regression tests > key d selects diamond tool > [end of test] number of renders 1`] = `6`;
+exports[`regression tests > key j selects diamond tool > [end of test] number of renders 1`] = `6`;
 
-exports[`regression tests > key d selects diamond tool > [end of test] redo stack 1`] = `[]`;
+exports[`regression tests > key j selects diamond tool > [end of test] redo stack 1`] = `[]`;
 
-exports[`regression tests > key d selects diamond tool > [end of test] undo stack 1`] = `
+exports[`regression tests > key j selects diamond tool > [end of test] undo stack 1`] = `
 [
   {
     "appState": AppStateDelta {
diff --git a/packages/excalidraw/tests/regressionTests.test.tsx b/packages/excalidraw/tests/regressionTests.test.tsx
index 986c828f..0974fc5e 100644
--- a/packages/excalidraw/tests/regressionTests.test.tsx
+++ b/packages/excalidraw/tests/regressionTests.test.tsx
@@ -143,7 +143,7 @@ describe("regression tests", () => {
 
   for (const [keys, shape, shouldSelect] of [
     [`2${KEYS.R}`, "rectangle", true],
-    [`3${KEYS.D}`, "diamond", true],
+    [`3${KEYS.J}`, "diamond", true],
     [`4${KEYS.O}`, "ellipse", true],
     [`5${KEYS.A}`, "arrow", true],
     [`6${KEYS.L}`, "line", true],
