diff --git a/packages/excalidraw/actions/actionCanvas.tsx b/packages/excalidraw/actions/actionCanvas.tsx
index 9c8fea32..cb88ce75 100644
--- a/packages/excalidraw/actions/actionCanvas.tsx
+++ b/packages/excalidraw/actions/actionCanvas.tsx
@@ -536,7 +536,11 @@ export const actionToggleEraserTool = register({
       captureUpdate: CaptureUpdateAction.IMMEDIATELY,
     };
   },
-  keyTest: (event) => event.key === KEYS.E,
+  keyTest: (event, appState) =>
+    event.key === KEYS.E &&
+    !appState.newElement &&
+    !appState.selectedLinearElement?.isEditing &&
+    !appState.selectedLinearElement?.isDragging,
 });
 
 export const actionToggleLassoTool = register({
